Misc/NEWS.d/next/Library/2018-04-22-21-42-31.bpo-25478.cMui8n.rst

```diff
@@ -0,0 +1 @@
+Adding scalar multiplication/division support to collections.Counter().
```

Comment:
Should `collections.Counter()` be `` :class:`collections.Counter()` ``?

Reply:
It's okay to make news entries without markup.